src/ResourceManager/Compute/Commands.Compute/Extension/DSC/SetAzureVMDscExtensionCommand.cs

```diff
@@ -240,6 +240,11 @@ public override void ExecuteCmdlet()
         //Private Methods
         private void ValidateParameters()
         {
+            if ("5.1PP".Equals(WmfVersion))
+            {
+                this.ThrowInvalidArgumentError(Microsoft.Azure.Commands.Compute.Properties.Resources.WMFVersionNotSupported, WmfVersion);
```

Comment:
@eshaparmar why was this change made? This is a breaking change - existing scripts that use this value for `WmfVersion` will no longer work. This is equivalent to removing the value from the `ValidateSet` attribute. 

If you want to make this change, you will need to change this error to a warning that tells customers that the value `5.1PP` is no longer supported and will be removed in an upcoming release. You will then need to document this in the [upcoming-breaking-changes.md](https://github.com/Azure/azure-powershell/blob/dev/src/ResourceManager/Compute/documentation/upcoming-breaking-changes.md) file.